frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -228,34 +226,43 @@ export default defineComponent({
 
     // Define computed properties dependent on the fee (must be computed to react to ETH gas price updates by user).
     // Variables prefixed with `formatted*` are inteded for display in the U)
-    const amountReceived = computed(() => amount.sub(useCustomFee.value ? customFeeInWei.value : fee.value)); // amount user will receive
-    const formattedFee = computed(() => {
-      if (isNativeToken) {
-        round(formatUnits(fee.value, decimals), ethDisplayDecimals);
-      } else {
-        return roundTokenAmount(props.activeFee.fee, props.activeFee.token);
-      }
-      return round(formatUnits(fee.value, decimals), numDecimals);
-    }); // relayer fee, rounded
+    const amountReceived = computed(() => amount.sub(useCustomFee.value ? customTxFeeInWei.value : fee.value)); // amount user will receive
+
+    // transaction fee, rounded
+    const formattedDefaultTxCost = computed(() => {
+      return humanizeTokenAmount(
+        (isNativeToken ? fee.value : props.activeFee.fee),
+        props.activeFee.token
+      )
+    });
```

Comment:
Nit, but maybe a small refactor for readability?

```suggestion
    // transaction fee, rounded
    const txCost = isNativeToken ? fee.value : props.activeFee.fee;
    const formattedDefaultTxCost = computed(() => humanizeTokenAmount(txCost, props.activeFee.token));
```